source/system/string/string_literal.h

```diff
@@ -0,0 +1,8 @@
+#ifndef __STRINGLITERAL_FIX_H__
+#define __STRINGLITERAL_FIX_H__
+
```

Comment:
That was the idea, I'm happy that you thought so. This is a hack and should be removed as soon as we get rid of String class.